common/extensions.go

```diff
@@ -122,11 +123,23 @@ func (bs ByteSliceExtension) RemoveBOM() []byte {
 
 /////////////////////////////////////////////////////////////////////////////////////////////////
 
+func DeterminePathSeparator(path string) string {
+	// Just use forward-slash everywhere that isn't windows.
+	if runtime.GOOS == "windows" && strings.Contains(path, `\`) {
```

Comment:
>runtime.GOOS == "windows" [](start = 4, length = 25)

How does this differ from `os.PathSeparator == '\\'`? I see both checks being used

Reply:
os.PathSeparator == '\\' is actually somewhat less strict than runtime.GOOS == "windows".

I don't _THINK_ there are any OSes other than windows that use a \ separator, and definitely none that we support.

For our use case, either would work. I've already changed the long path handler to use runtime.GOOS instead since I would rather be safer than sorry.